index.js

```diff
@@ -1,4 +1,6 @@
 exports.handler = function(sns_event, context, callback) {
+  process.env.PATH = `${process.env.PATH}:${process.env.LAMBDA_TASK_ROOT}`;
```

Comment:
Where does this env variable come from? Why is it concatenated in the code and not when the PATH env is set?

Reply:
Ok... seems weird why the PATH variable should affect our code, but whatevs :D